docs/nddata/index.rst

```diff
@@ -156,6 +168,47 @@ Code that uses the arithemtic methods that used to be included in
 instead subclass `~astropy.nddata.NDDataArray`; that class is equivalent to
 the original `~astropy.nddata.NDData` class.
 
+.. _nddata_transition_2:
+
+Transition to astropy xx
+========================
+
+The `~astropy.nddata` package was subject to another revision mainly to fix
+existing bugs but also to implemented new functionality while trying to keep
+the previous API or only to extend it.
+
+These changes included:
+
++ `~astropy.nddata.NDDataBase` now enforces no restrictions. Previous
+  implementations forced
+  the ``uncertainty`` to match certain requirements, these were dropped.
++ `~astropy.nddata.NDData` underwent several  bugfixes and documentation
+  updates. But several
+  small new functionalities are now included like creating an instance now
+  allows for a ``copy`` parameter (default is ``False``) and is less
+  restrictive on the ``uncertainty`` (only a Warning is issued if the
+  uncertainty does not meet the requirements instead of an Exception).
++ `~astropy.nddata.NDSlicingMixin` is now more restrictive while slicing
+  ``meta`` attributes
+  like ``uncertainty``, ``wcs`` and ``mask``. But the Mixin was refactored in
+  a way that allows subclasses to customize slicing attributes in a different
+  way. Visit :ref:`nddata/subclassing` for more information.
```

Comment:
one of the sphinx warning is that there is no such ``nddata/subclassing`` label, you should define it somewhere.